package.json

```diff
@@ -15,7 +15,9 @@
       "prettier --write"
     ],
     "*.py": [
-      "python3.7 -m pipenv run black"
+      "python3.7 -m pipenv run black",
+      "python3.7 -m pipenv run pylint",
+      "make typecheck-server"
```

Comment:
Seems like this will be called like e.g. `make typecheck-server app.py util/binpacking.py`, which is probably not what we want. Are you just trying to add a `pre-commit` step?

Reply:
Yes, I'm just trying to add a pre-commit step. I tried `python3.7 -m pipenv run mypy` but mypy didn't like receiving the files independently, it wanted to run on the whole app at once. I figured `make typecheck-server` would ignore the rest of the args so it would work, even though it's not ideal. Any other ideas?

Reply:
You could just add it to the `husky.hooks.pre-commit` command in this file.